ndk-glue/src/lib.rs

```diff
@@ -16,6 +16,18 @@ use std::thread;
 
 pub use ndk_macro::main;
 
+/// `ndk-glue` macros register the reading end of an event pipe with the
+/// main [`ThreadLooper`] under this ident `ident`.
+/// When returned from [`ThreadLooper::poll_*`](ThreadLooper::poll_once)
+/// an event can be retrieved from [`poll_events()`].
```

Comment:
I really wanted to have newlines here, but that makes the second sentence invisible in the documentation unless explicitly clicking on the constant :disappointed: